hack/scan-image-vuln.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+# Copyright 2024 The Karmada Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# This script starts a images scanning with trivy
+# This script depends on utils in: ${REPO_ROOT}/hack/util.sh
+# 1. used to locally scan Karmada component image vulnerabilities with trivy
+# 2. Used to scan specified image with trivy
+
+function usage() {
+    echo "Usage:"
+    echo "    hack/vuln-scan-local.sh [-i imageRef] [-r registry] [-v version] [-s skip-image-generation] [-h]"
```

Comment:
```suggestion
    echo "    hack/scan-image-vuln.sh [-i imageRef] [-r registry] [-v version] [-s skip-image-generation] [-h]"
```

Reply:
Can you give an example of how to run this script?
Do we need the -r -v? Is an image reference enough?

Reply:
> Do we need the -r -v? Is an image reference enough?

I think we need this. Because we hope that this script can not only scan the latest version of the image, but also scan old versions or customized versions. An image referenc can only scan one image at the same time.

Reply:
> Can you give an example of how to run this script? 

I add it to the usage of this script